lstchain/image/modifier.py

```diff
@@ -53,33 +56,59 @@ def add_noise_in_pixels(rng, image, extra_noise_in_dim_pixels,
 
     return image
 
+@njit(cache=True)
+def set_numba_seed(seed):
+    np.random.seed(seed)
 
-def smear_light_in_pixels(image, camera_geom, smeared_light_fraction):
+@njit(cache=True)
+def random_psf_smearer(image, fraction, indices, indptr):
     """
-
     Parameters
     ----------
     image: charges (p.e.) in the camera
 
-    camera_geom: camera geometry
+    indices: camera_geometry.neighbor_matrix_sparse.indices
+
+    indptr: camera_geometry.neighbor_matrix_sparse.indptr
```

Comment:
what is `indptr`? can we include a more explicit name for it (maybe also change it in ctapipe)?

Reply:
It tells you where the indices of a given pixel's neighbors are in the array  neighbor_matrix_sparse.indices

Reply:
`intptr` and `indices` are the two arrays of a `scipy.sparse.csr_matrix` that tell you which values are actually non-zero.

These names won't change if scipy doesn't break everyones code.